app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1558,8 +1734,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
     
     private void createPreferencesMenu() {
-        Intent i = new Intent(this, CommCarePreferences.class);
-        CommCareHomeActivity.this.startActivityForResult(i, PREFERENCES_ACTIVITY);
+        createPreferencesMenu(this);
+    }
+
+    public static void createPreferencesMenu(Activity activity) {
```

Comment:
Maybe rename to 'launchPreferencesMenu' or comment it so that it is apparent that it launches something.